examples/vector_return/src/main.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python2
```

Comment:
Thanks for adding this, as it made me realize I had a bunch of old junk! #45 updates this to `python3` and adds a space; I'd appreciate it if you could match.